src/SPC/command/DumpExtensionsCommand.php

```diff
@@ -4,67 +4,157 @@
 
 namespace SPC\command;
 
+use SPC\store\FileSystem;
 use Symfony\Component\Console\Attribute\AsCommand;
-use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
 
 #[AsCommand(name: 'dump-extensions', description: 'Determines the required php extensions')]
 class DumpExtensionsCommand extends BaseCommand
 {
-    private array $files = [
-        'vendor/composer/installed.json',
-        'composer.lock',
-        'composer.json',
-    ];
+    protected bool $no_motd = true;
 
-    public function handle(): int
+    public function configure(): void
     {
-        $fs = new Filesystem();
```

Comment:
Filesystem class is in spc's dev packages, we cannot use it when spc is built as standalone binary.